apps/src/applab/DesignToggleRow.jsx

```diff
@@ -62,7 +62,7 @@ module.exports = React.createClass({
       verticalAlign: 'top',
       border: '1px solid #949ca2',
       margin: '0 0 8px 0',
-      padding: '3px 6px',
+      padding: '2px 6px',
```

Comment:
Had to make the design toggle a little smaller to not look crowded (since the visualization moves up to line up with the bottom of the workspace header).

![smaller_toggle](https://cloud.githubusercontent.com/assets/1615761/8135281/f535fe6c-10ea-11e5-8550-97dba97acf8e.png)